.develop.sh

```diff
@@ -13,6 +13,16 @@ VERSION=$1
 # The version to clone
 LATESTVERSION=`ts-node src/scripts/latestVersion.ts`
 
+# The oldest version still supported
+LASTVERSION=`ts-node src/scripts/oldestVersion.ts`
```

Comment:
In a future pass over these scripts, we might want to standardize the name to `oldestVersion`. I think is not only here but in some other places.